src/components/chip/chip.tsx

```diff
@@ -0,0 +1,279 @@
+import {
+    Component,
+    Element,
+    Event,
+    EventEmitter,
+    h,
+    Prop,
+} from '@stencil/core';
+import { Icon, Languages, Link } from '../../interface';
+import { getIconName } from '../icon/get-icon-props';
+import {
+    makeEnterClickable,
+    removeEnterClickable,
+} from '../../util/make-enter-clickable';
+import translate from '../../global/translations';
+import {
+    BACKSPACE,
+    BACKSPACE_KEY_CODE,
+    DELETE,
+    DELETE_KEY_CODE,
+} from '../../util/keycodes';
+
+/**
+ * Chips and buttons are both interactive elements in UI design,
+ * but they serve different purposes and are used in different contexts.
+ *
+ * :::warning
+ * Do not use the chip component, carelessly as an alternative for
```

Comment:
```suggestion
 * Do not use the chip component carelessly, as an alternative for
```

`fixup! chore(chip): add new private component`